src/cpp/core/gwt/GwtFileHandler.cpp

```diff
@@ -133,11 +133,19 @@ void handleFileRequest(const FileRequestOptions& options,
                          (request.queryParamValue("emulatedStack") == "1");
       vars["compiler_stack_mode"] = useEmulatedStack ? "emulated" : "native";
 
+      // polyfill for IE11 (only)
+      std::string polyfill = "<script type=\"text/javascript\" language=\"javascript\" src=\"js/core-js/minified.js\"></script>\n";
+      if (regex_utils::match(request.userAgent(), boost::regex(".*Trident.*"))) {
+         vars["head_tags"] = polyfill;
+      } else {
+         vars["head_tags"] = std::string();
+      }
+
       // check for initJs
       if (!options.initJs.empty())
-         vars["head_tags"] = "<script>" + options.initJs + "</script>";
+         vars["head_tags"] = vars["head_tags"] + "<script>" + options.initJs + "</script>";
       else
-         vars["head_tags"] = std::string();
+         vars["head_tags"] = vars["head_tags"] + std::string();
```

Comment:
This looks like a no-op -- should we just omit the else?

Reply:
Good catch, thanks!